text/0000-custom-element.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2019-07-05
+- RFC PR: (leave this empty)
+- Lightning Web Component Issue: (leave this empty)
+
+# Summary
+
+Facilitate the registration of LWC Constructor as Custom Elements.
+
+# Motivation
+
+As today, the way to generate a qualifying Custom Element from a LWC Constructor, is via an experimental API: `buildCustomElementConstructor()`. Since this is a primary use case for LWC, we want to make it more ergonomic, and simpler. Additionally, the experimental API allow you to generate more than one Custom Element per LWC Constructors, which does not align with the web components semantics.
+
+# Goals
+
+1. Provide a dead simple way to register a custom element from a LWC Constructor.
+2. Preserve the web components semantics.
+
+# Proposal
+
+This proposal provides a high-level API (an abstraction layer) for authors to access the custom element associate to a LWC Constructor:
+
+```js
+import XFoo from 'x/foo';
+customElements.define('x-foo', XFoo.CustomElement);
```

Comment:
this is the meat of this proposal... the `.CustomElement` static accessor on every LWC constructor

Reply:
The main issue that I have with `CustomElement`, is that the custom element represents a custom element instance while the custom elements registry accepts a constructor. I support @apapko into making this explicit.

Reply:
I mentioned a similar concern in https://github.com/salesforce/lwc/pull/1395#pullrequestreview-259747574